예산.js

```diff
@@ -0,0 +1,14 @@
+function solution (d, budget) {
+    let answer = 0, sum = 0;
```

Comment:
변수 선언을 ,로 해주는게 좋아보이진 않아요
분리해서 선언 해주는게 좋을 듯 합니다
```suggestion
    let answer = 0
    let sum = 0;
```